jekyll.gemspec

```diff
@@ -41,10 +41,10 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency("kramdown",              "~> 2.3")
   s.add_runtime_dependency("kramdown-parser-gfm",   "~> 1.0")
   s.add_runtime_dependency("liquid",                "~> 4.0")
-  s.add_runtime_dependency("mercenary",             "~> 0.4.0")
```

Comment:
This downgrades the version from `>= 0.4.0 && < 0.5.0` to `>= 0.3.6 && < 0.4.0`. More about the `~>` pessimistic constraint can be found here:

https://guides.rubygems.org/patterns/#pessimistic-version-constraint

Reply:
Thank you for pointing this out! I mixed this with some other semver specification language, npm's probably. Changed to `"~> 0.3", ">= 0.3.6"` and tested that it can select 0.4.0.

Reply:
Yes, this could work, if also 0.5 is acceptable for upstream

Reply:
Hmm, that was not my intent either. Simply to allow 0.3.6 and not change anything else. I am pessimistic about my ability to use the pessimistic constraint correctly. Better to stay with standard mathematical operators here. One last try: `">= 0.3.6", "< 0.5"`.